oaknational/Oak-Web-Application
# LESQ-880: Lesson Listing page slide decks

## Description

Music year: 1964

- Removes hardcoded false value and returns bool depending on if copyright content is available or not

## Issue(s)

Fixes #LESQ-880

## How to test

1. Go to https://deploy-preview-2534--oak-web-application.netlify.thenational.academy
2. Click on MFL languages which containt copyright 
3. You should see shouldn't display slide deck as available
4. All other lessons should be un affected

## Screenshots

How it used to look (delete if n/a):
<img width="1024" alt="Screenshot 2024-06-25 at 11 00 46" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/151c202a-cea2-439d-b778-7422a4de667f">

How it should now look:

<img width="1014" alt="Screenshot 2024-06-25 at 11 00 05" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/2134f7e5-c75d-4e4b-ba9e-e7208d422646">


## Checklist
